src/Command/Resources/ResourcesSetCommand.php

```diff
@@ -231,7 +237,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             // Set the instance count.
             if (!$service instanceof Service) { // a Service instance count cannot be changed
-                if (isset($givenCounts[$name])) {
+                if (!empty($autoscalingEnabled[$name])) {
+                    $this->stdErr->writeln(sprintf('The instance count of the service <error>%s</error> cannot be changed when autoscaling is enabled.', $name));
```

Comment:
```suggestion
                    if (isset($givenCounts[$name])) {
                        $this->stdErr->writeln(sprintf('The instance count of the service <error>%s</error> cannot be changed when autoscaling is enabled.', $name));
                        $errored = true;
                    }
```

I'm not sure if we need a message here when nothing is specified - what do you think?

(And if something was specified, we should set `$errored`)